tests/Feature/Flights/ListFlightsTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Flights;
+
+use Database\Factories\FlightFactory;
+
+use function Pest\Laravel\getJson;
+
+test('can list flights successfully', function () {
+    FlightFactory::times(5)->create();
+
+    $response = getJson('/flights');
```

Comment:
```suggestion
    $response = get('/flights');
```
Using `getJson` would be helpful when you're making a request to an endpoint that you'd expect to respond with a JSON, but the `/flights` route returns a view not a JSON.